### DISCUSSION TITLE
SSD1306 and Metadata

### DISCUSSION THREAD

**Discussion by RuloH** [pschatzmann/ESP32-A2DP#255] [Q&A] [answered]
Hello! First of all I want to tank you for your code. awesome.

I have been trying to print the metadata (a2dp_sink.set_avrc_metadata_callback(avrc_metadata_callback)) into an OLED SSD1306 using the adafruit library and I can't figure out how to get the data2 into the void loop. any suggestion? 

Thank you very much in advance. Greetings and best regards.

**Comment by clarkbremer** [comment, marked as answer]
Don't try to print anything in the callback.  Just stash the data in a variable, and print it out in code in the loop.

**Reply by msaraswat15031997** [reply]
Hello sir, can you show me an example by code on how to stash the data please? Much appreciated thanks
